live-examples/js-examples/date-parse.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">var unixTimeZero = Date.parse('Thu, 01 Jan 1970 00:00:00 GMT');
+var javaScriptRelease = Date.parse('04 Dec 1995 00:12:00 GMT');
+var millisElapsed = javaScriptRelease - unixTimeZero;
+
+console.log(millisElapsed);
+//expected output: 818035920000
```

Comment:
I think it's a bit strange doing arithmetic here when `unixTimeZero` === 0, so `javaScriptRelease` === `millisElapsed`.

I know it's boring, but I think it would be fine to just log `unixTimeZero` and  `javaScriptRelease` here.

Reply:
OK